src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,39 @@
+package duke.task;
+
+/**
+ * Represents a task to be added by the user.
+ */
+
+public class Task {
+    protected String name ;
+    protected boolean status ;
```

Comment:
```suggestion
    protected boolean isCompleted;
```
Boolean variables/methods should be named to sound like booleans